ui/components-react/src/components-react/filter-builder/FilterBuilderRule.scss

```diff
@@ -45,10 +26,22 @@
     width: 15rem;
     display: flex;
     flex-direction: column;
+
+    >span {
+      .components-text-editor {
+        height: unset;
+      }
+    }
   }
 
   .fb-row-toolbar {
     width: var(--toolbar-width);
     margin-left: auto;
+    justify-content: space-between;
+    height: 26px;
+
+    >button {
+      height: 100%;
+    }
```

Comment:
This will be fixed in ITwinUI so you can revert changes regarding button size and placement https://github.com/iTwin/iTwinUI/discussions/2280#discussioncomment-10831762